pkg/search/postgres/query/query_functions.go

```diff
@@ -49,31 +49,38 @@ var datatypeToQueryFunc = map[walker.DataType]queryFunction{
 	// Map is handled separately.
 }
 
-func matchFieldQuery(dbField *walker.Field, field *pkgSearch.Field, value string, highlight bool, now time.Time) (*QueryEntry, error) {
-	qualifiedColName := dbField.Schema.Table + "." + dbField.ColumnName
-
+func matchFieldQuery(qualifiedColName string, dataType walker.DataType, field *pkgSearch.Field, value string, highlight bool, now time.Time, goesIntoHavingClause bool) (*QueryEntry, error) {
```

Comment:
is `dataType` sql type? Maybe expand the var name?

Reply:
Yup, done.